Agents/DerivationAgentPythonExample/docker-compose-test-dockerised-debug.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.8'
+
+services:
+  derivation_agent_python_example_debug:
+    image: ghcr.io/cambridge-cares/derivation_agent_python_example_debug:1.0.0-SNAPSHOT
+    container_name: derivation_agent_python_example_debug
+    environment:
+      LOG4J_FORMAT_MSG_NO_LOOKUPS: "true"
+      EMAIL_AUTH_JSON_PATH: /run/secrets/email_auth
+    build:
+      context: .
+      target: test_dockerised_debug
+      args:
+        # Build argument used to construct paths in Dockerfile (needs to match name in setup.py)
+        agentdir: derivationagentpythonexample
+    ports:
+      - 7000:5000
```

Comment:
Have you successfully tested the debugging image? I am not sure how the debugger shall attached to the container without additional port clarification here, i.e. 5678? Furthermore, should we add a vscode launch configuration to attach to the debug test image? Without it, we might make people to go through a lot of avoidable pain.

Reply:
Addressed in https://github.com/cambridge-cares/TheWorldAvatar/pull/391/commits/6441b076621f2437e17c65a647f979b11d063495